src/scripts/seedDB.js

```diff
@@ -0,0 +1,171 @@
+require("dotenv").config({ silent: true });
+
+const { DB_URL } = process.env;
+
+// db dependencies
+const mongoose = require("mongoose");
+const uuid = require("uuid");
+
+// validation dependencies
+const { isValid, format } = require("date-fns");
+const bcrypt = require("bcryptjs");
+const { convert } = require("emoji-text");
+
+// scraping dependecies
+const cheerio = require("cheerio");
+const axios = require("axios");
+
+const fetch_url = "https://mlh.io/seasons/na-2019/events";
+const seedInstance = axios.create({ baseURL: fetch_url });
+
+/** START DB Connection */
+if (!DB_URL) {
+  console.log("> 🙅🏻‍♂️ DB URL not provided");
+  process.exit(0);
+}
+
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useFindAndModify: false
+};
+
+async function db() {
+  Promise.resolve(mongoose.connect(DB_URL, options));
+}
+/** END DB Connection */
+
+/** Mongoose schema definition */
+const Schema = mongoose.Schema;
+
+const eventSchema = new Schema(
+  {
+    _id: {
+      type: String,
+      binData: Buffer,
+      index: true,
+      unique: true,
+      required: true,
+      default: uuid.v4
+    },
+    name: { type: String, required: true },
+    date: { type: Date, required: true },
+    mainImageUrl: { type: String, required: true },
+    mainColor: { type: String, default: "#6870FF" },
+    logoUrl: { type: String, required: true },
+    secretKey: { type: String, required: true },
+    slug: { type: String }
+  },
+  { timestamps: true, autoIndex: false }
+);
+
+const Event = mongoose.model("Event", eventSchema);
```

Comment:
I think it's a bit weird that this knows about the model because it requires us to duplicate a lot of the same db logic and the models, and they could go out of sync. What if we make API calls to create the event instead of going behind the scenes? That would also give us some sort of 'tests' that our API works as expected.